tests/trace_server/conftest_lib/clickhouse_server.py

```diff
@@ -103,7 +103,7 @@ def ensure_clickhouse_db_process_running(
 
     The fixture handles cleanup by stopping the process when the test session ends.
     """
-    server_up = check_server_up(host, port, 0)
+    server_up = check_server_up(host, port, 1)
```

Comment:
why this change?

Reply:
0 was a bug - 0 doesn't wait at all, 1 tries once